### ISSUE TITLE
The problem on Go To Definition with special words (Given When And Then)

### ISSUE THREAD

**Issue by jetstreamc** [tomi/vscode-rf-language-server#9]
The "Go To Definition" feature does not support the statements with "Given", "When", "And" and "Then" keyword. For example,
```
Verify leave all input to blank
    Given admin want to create a product
    When admin leave all input to blank
    And admin click save button
    Then warning message will be displayed
```
However, when removing these special keywords from the statements, The Go To Definition works fine.

**Comment by tomi** [comment]
Thank you for filing an issue. Test cases written in behavior-driven style (Given-When-Then) are not yet supported, but it's something that shouldn't be too hard to implement.

**Comment by cisum-la** [comment]
Is Given-When-Then on the roadmap for upcoming releases

**Comment by brianon75** [comment]
I just installed this extension. It seems great except I cannot use without the Gherkin support. Would really be the icing on the cake.

**Comment by roryjmaher** [comment]
Just wondering on creating a PR for something like this? It looks like if we attempted to match the keyword ignoring a list of given cucumber keywords it might do the job?

**Comment by tomi** [comment]
I haven't personally been using the behavior-driven style, so this hasn't been on the top of my list. If someone wants to give a try to implement this, please go ahead. I'm just thinking if the Given-When-Then etc. should always be parsed as special labels or if this is something that should be configurable..?

**Comment by bergstenarn** [comment]
I'm also interested in using this VSCode extension but the lack of Gherkin support is a blocker. I wouldn't mind implementing Gherkin support myself. However I have absolutely no experience in coding VSCode extensions, so I would probably just mess up if I tried ;-)

**Comment by tomi** [comment]
I'm happy to announce that this feature is included the newest release 2.4.0 :)
